roles/validations/tasks/xmlcreate.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Get validation start time from system
+  ansible.builtin.command:
+    cmd: date +%Y-%m-%d-%H:%M:%S.%3N
+  register: individual_validation_start_time
+
+- name: Run a validation
+  ansible.builtin.include_tasks: "{{ item }}"
```

Comment:
A task file called xmlcreate.yml that runs a validation task is a bit confusing, it does way more than generating an XML.

Reply:
Renamed to run_validation.yml